### DISCUSSION TITLE
Mac High Sierra coapp is not being rcognized, deleting and reinsatalling isnt helping.I have posted what I got from terminal.

### DISCUSSION THREAD

**Discussion by AKOsiris** [aclap-dev/video-downloadhelper#193] [Help and Q&A]
### Your question

I am pretty much a noob when it comes to computers but maybe my issue is the same as the Linus usr/ one Idk, I have pasted the coapp details I got from the firefox extension. I have tried both pkg and dmg installers. In the first case the installation fails due to unexpected error and it says to contact the software developer and in the second case the coapp seems to open and close for the first time but no coapp installed and subsequent tries to open the coapp from the apps seems to do nothing. This is what I got from Terminal 
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

Abort trap: 6
Any help is appreciated

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: nocoapp
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: no.

**Comment by paulrouget** [comment]
High Sierra is a little too old. Can you upgrade to MacOS 10.14 or above?

**Reply by AKOsiris** [reply]
Is there nothing that can be done for Big Sur? I am planning to update to Catalina/ mojave but I am scared of bricking my mbp its from 2015.

**Reply by paulrouget** [reply]
You can downgrade back to the old version of VDH: https://github.com/aclap-dev/video-downloadhelper/wiki/Older-versions

**Reply by AKOsiris** [reply]
Can you tell me whats the last compatible version with Big Sur ? for both the vdh & coapp, I couldnt find the details on mozilla for compatibility.

**Reply by 2Config7** [reply]
Same here. Stopped working on High Seirra and reinstalling old version is not working either. Always getting prompt to ugrade even though upgade version doesn't work.

**Reply by Mattmjs** [reply]
Big Sur is version 11 of Mac OS according to a web search, so the companion app should work.

**Reply by kit529738** [reply]
Hi-- I'm using it fine on High Sierra, os ver 10.13, [ please see my response here ](https://github.com/aclap-dev/video-downloadhelper/discussions/288#discussioncomment-8966571)
